### ISSUE TITLE
The settings for KNN classifier?

### ISSUE THREAD

**Issue by Hzzone** [salesforce/PCL#13]
Thanks for your work. I am confused about your settings for KNN classifier.
I used the KNN predict function from MoCo, which uses the test set to match the training set with K nearest neighbors and T for temperature.
I am curious that, what K and T do you set, and which train or test set do you use as probe set?
Looking forward to your reply!

**Comment by LiJunnan1992** [comment]
Hi, I used K=20 and T=0.07. I used the test set as query to search for nearest neighbors in the training set.

**Comment by Hzzone** [comment]
Thanks for your reply.
